doc/content/reference/api/_index.md

```diff
@@ -4,3 +4,15 @@ description: ""
 ---
 
 This is the reference for the gRPC and HTTP APIs that {{% tts %}} exposes.
+
+<!--more-->
+
+## Field Masks
+
+{{% tts %}} APIs use field masks to specify a subset of fields that should be returned by a `Get` request, or to specify fields that should be updated in an `Update` request. See Google's [protocol buffer reference](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.FieldMask) for more information about field masks.
```

Comment:
```suggestion
{{% tts %}} APIs use field masks to specify a subset of fields that should be returned by a reading request, or to specify fields that should be updated in a writing request. See Google's [protocol buffer reference](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.FieldMask) for more information about field masks.
```

It's not only `Get`; it can also be listing, searching etc. Same for `Update`; can be setting stuff.

Reply:
Also, Protocol Buffers is a name so title case